src/index.js

```diff
@@ -55,6 +56,7 @@ class _App extends Component {
             <Route exact path="/users/:id" component={User} />
             {loggedIn && <Redirect to="/" />}
             <Route path="/login" component={Login} exact />
+            <Route exact path="/admin" component={Admin} />
```

Comment:
If I'm correct, the admin path will redirect to '/' since it is below ```{loggedIn && <Redirect to="/" />}``` Just hoist it above that.